sympy/physics/mechanics/body.py

```diff
@@ -99,7 +99,6 @@ class Body(RigidBody, Particle):  # type: ignore
     def __init__(self, name, masscenter=None, mass=None, frame=None,
                  central_inertia=None):
```

Comment:
Do we want to add any sort of deprecation note on instantiation?

Reply:
`Body` is not deprecated yet in this PR, that will only be done after the joints framework has been made fully compatible with `Rigidbody` and `Particle`. Also would like to add the deprecation in a separate PR, because quite some documentation will have to be changed as well.